src/ensembl/src/content/app/custom-download/state/filters/filterState.ts

```diff
@@ -0,0 +1,17 @@
+import { Attributes } from 'src/content/app/custom-download/types/Attributes';
+import JSONValue from 'src/shared/types/JSON';
+import customDownloadStorageService from 'src/content/app/custom-download/services/custom-download-storage-service';
+
+export type FilterAccordionState = Readonly<{
```

Comment:
Same as with AttributeAccordionState — rename to FilterState

Reply:
That's done